docs/guidelines/developers/handling-tickets.md

```diff
@@ -1,27 +1,27 @@
 ---
 title: Handling Tickets
-description: Guidelines for managing and resolving achievement tickets, including steps for addressing your own tickets, handling tickets for others, and protocols for closing tickets based on various criteria.
+description: Guidelines for managing and resolving achievement tickets, including steps for addressing one's own tickets, handling tickets for others, and protocols for closing tickets based on various criteria.
 ---
 
 # Handling Tickets
 
 [[toc]]
 
-## Handling Your Own Tickets
+## Handling Own Tickets
 
-**After you've published your achievements, be prepared for bug reports.**
-You're expected to keep your work bug free by appropriately resolving tickets. Respond to all tickets as soon as possible. The sooner you respond the better, because the problem is fresh in the player's mind and you can use them to help you to resolve the problem.
+**After a developer has published achievements, they should be prepared for bug reports.**
+A developer's top priority on RetroAchievements is to keep their work bug free by appropriately resolving tickets. Developer shall respond to all tickets as soon as possible. The sooner the response, the better, because the problem is fresh in the player's mind and their knowledge can be used to help resolve the problem.
```

Comment:
```suggestion
A developer's top priority on RetroAchievements is to keep their work bug free by appropriately resolving tickets. A developer shall respond to all tickets as soon as possible. The sooner the response, the better, because the problem is fresh in the player's mind and their knowledge can be used to help resolve the problem.
```